LondonBritishLibrary/orient/BLorient550.xml

```diff
@@ -5,7 +5,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title>Maṣḥafa faws manfasāwi (Michael of ʾAṯrīb), Maṣḥafa taklil, Maṣḥafa qandil, Śǝrʿāt za-ʾagbǝʾo tābota wǝsta beta krǝstiyān, Maṣḥafa qedr,
+                <title>Maṣḥafa faws manfasāwi (Michael of ʾAṯrīb), Maṣḥafa taklil, Maṣḥafa qandil, Śǝrʿāt za-ʾagbǝʾo tābot wǝsta beta krǝstiyān, Maṣḥafa qedr,
```

Comment:
I actually find the original Collection of service books nicer as a short title - the precise content is visible later, but it is not so important